.pipelines/multitenancy/swiftv2-e2e-step-template.yaml

```diff
@@ -0,0 +1,85 @@
+parameters:
+  name: ""
+  clusterName: ""
+  cni: cniv2
+  os: ""
+
+steps:
+  - bash: |
+      go version
+      go env
+      mkdir -p '$(GOBIN)'
+      mkdir -p '$(GOPATH)/pkg'
+      mkdir -p '$(modulePath)'
+      echo '##vso[task.prependpath]$(GOBIN)'
+      echo '##vso[task.prependpath]$(GOROOT)/bin'
+    name: "GoEnv"
+    displayName: "Set up the Go environment"
+
+  - task: KubectlInstaller@0
+    inputs:
+      kubectlVersion: latest
+
+  - task: AzureCLI@1
+    inputs:
+      azureSubscription: $(ACN_TEST_SERVICE_CONNECTION)
+      scriptLocation: "inlineScript"
+      scriptType: "bash"
+      addSpnToEnvironment: true
+      inlineScript: |
+        set -e
+        make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}
+        ls -lah
+        pwd
+        kubectl cluster-info
+        kubectl get po -owide -A
+        echo "Apply the pod network yaml to start the delegation"
+        less test/integration/manifests/swiftv2/podnetwork.yaml
+        envsubst '${SUBNET_TOKEN},${SUBNET_GUID},${SUBNET_RESOURCE_ID},${VNET_GUID}' < test/integration/manifests/swiftv2/podnetwork.yaml | kubectl apply -f -
+        echo "Check the podnetwork yaml file"
+        less test/integration/manifests/swiftv2/podnetwork.yaml
+        kubectl get pn
+        kubectl describe pn
+        echo "Apply the pod network instance yaml to reserve IP"
+        kubectl apply -f test/integration/manifests/swiftv2/pni.yaml
+        kubectl get pni
+        kubectl describe pni
+        export NODE_NAME_0="$(kubectl get nodes -o json | jq -r .items[0].metadata.name)"
+        echo $NODE_NAME_0
+        echo "Start the first pod using the reserved IP"
+        envsubst '$NODE_NAME_0' < test/integration/manifests/swiftv2/mtpod0.yaml | kubectl apply -f -
+        export NODE_NAME_1="$(kubectl get nodes -o json | jq -r .items[1].metadata.name)"
+        echo $NODE_NAME_1
+        echo "Start another pod using the reserved IP"
+        envsubst '$NODE_NAME_1' < test/integration/manifests/swiftv2/mtpod1.yaml | kubectl apply -f -
+        sleep 2m
+        kubectl get pod -o wide -A
+        sleep 2m
+        echo "Check pods after 4 minutes"
+        kubectl get po -owide -A
+        kubectl describe pni
+        echo "Start the connection test"
```

Comment:
lets move this part to datapath_swiftv2_test.go and grab IP from mtpnc instead of pni

Reply:
Could we use the script for now and move to datapath_swiftv2_test.go later? I have the code ready but I need time to test it. We can't test that in local so it's time consuming.
Used mtpnc instead of pni.